src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContext.cs

```diff
@@ -105,10 +105,10 @@ public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly)
                 }
                 else if (!resourceAssembly.IsDynamic)
                 {
-                    // EscapedCodeBase won't be supported by emitted assemblies anyway
+                    // Location won't be supported by emitted assemblies anyway
```

Comment:
Also not sure if this is true

Reply:
This also needs a check for Assembly.Location returning empty string - same reasoning as above.

Reply:
Updated based on feedback